libp2pdht/private/eth/p2p/discoveryv5/protocol.nim

```diff
@@ -1023,76 +1030,6 @@ func init*(
     bitsPerHop: bitsPerHop
   )
 
-proc newProtocol*(
```

Comment:
Found this unused newProtocol method.
(Looks like it could be an old copy-paste example that was never removed.)